libminifi/include/utils/gsl.h

```diff
@@ -26,6 +28,26 @@ namespace minifi {
 
 namespace gsl = ::gsl_lite;
 
+namespace utils {
+namespace detail {
+template<typename T>
+using remove_cvref_t = typename std::remove_cv<typename std::remove_reference<T>::type>::type;
```

Comment:
`gsl_lite` already has a `remove_cvref`, I do favor this version, we could change the `gsl_lite` one in `optional_from_ptr` to this one

Reply:
changed to `detail::remove_cvref_t` from our own gsl.h in f936b60